### ISSUE TITLE
ResourcesBot: Write report to mediawiki (language summary page)

### ISSUE THREAD

**Issue by holybiber** [4training/pywikitools#48]
We introduce new internal language summary pages for translators / translation coordinators, following the example of https://www.4training.net/4training:TestLanguage
ResourcesBot should update these pages automatically if there are any changes.

Implement as a GlobalPostProcessor as we need to read from the English LanguageInfo object as well (to be able to compare versions).
The WriteLists module has a similar functionality: create the mediawiki string, find its position in the whole page, replace if there are changes.

Coloring details:
- Version is either green (same as English) or red
- If version is red, no other cell can be green. It can be either orange (something is already done) or red (nothing done yet)
- If progress is unfinished (by the definition of TranslationProgress.is_unfinished()), progress should be red. If it's incomplete, progress should be orange. If it's 100% progress can be green (if version is green as well)
- PDF and ODT fields are red if those files are missing. If they exist, their color is green or orange, depending on the color of version.
- The first two columns are red if all the rest is red, green if all the rest is green and orange for all other cases

Depends on #47 
#36 is probably not necessary anymore afterwards
